### PR TITLE
Feature Contact Information Entity

### PR DESCRIPTION
**Pull request by Jugklgl**

### CODE REVIEW

File: src/main/java/com/example/rets_api/entity/PhoneEntity.java [hunk 1]
```diff
@@ -20,22 +18,22 @@ public class PhoneEntity {
     private String primaryPhone;
     private String alternatePhone;
 
-    @OneToOne(mappedBy = "agentPhone")
+    @OneToOne(mappedBy = "agentPhone", cascade = CascadeType.ALL)
```
Comment (Jugklgl) [inline src/main/java/com/example/rets_api/entity/PhoneEntity.java:21]: I added the 'cascade' in all fields to resolve an exception when I ran the tests.

File: src/test/Rets_Api.postman_collection.json [hunk 2]
```diff
@@ -194,6 +194,18 @@
...
+							"value": "5145556666",
+							"disabled": true
+						},
+						{
+							"key": "phoneNumbers",
+							"value": "5147778888",
+							"disabled": true
+						},
+						{
+
```
Comment (taisemarques) [inline src/test/Rets_Api.postman_collection.json:208]: Please update the request body of "Create Property with all Fields"

File: src/test/java/com/example/rets_api/repository/PropertyRepositoryJPATest.java [hunk 3]
```diff
@@ -86,10 +82,16 @@ public void should_store_a_PropertyAndSchoolAndRoomAndViewData() {
         assertNotNull(propertySaved.getSchoolList().get(0).getSchoolId());
         assertNotNull(propertySaved.getViewData().getViewDataId());
 
+
+        assertNotNull(propertySaved.getContactInformation());
```
Comment (taisemarques) [inline src/test/java/com/example/rets_api/repository/PropertyRepositoryJPATest.java:86]: Please add an assert to check contactInformationID

File: src/test/Rets_Api.postman_collection.json (hunk 2 above)
Comment (Jugklgl) [reply src/test/Rets_Api.postman_collection.json:208]: done in last commit

**Review by taisemarques — APPROVED** [review]

**Review by nazliBeit — APPROVED** [review]

**Review by sandy-ja — APPROVED** [review]

**Pull request merged**